auction9-frontend/src/app/data-table/data-table.component.ts

```diff
@@ -29,32 +30,24 @@ export interface AuctionData {
 })
 
 export class DataTableComponent implements AfterViewInit {
+  // data from parent
+  @Input() auctionsArray : AuctionData[];
 
   displayedColumns: string[] = ['auctionID', 'title', 'price', 'action'];
```

Comment:
Let's pass array of table headers also from Parent.

Reply:
Changed that 'home' pass table headers to data-table